src/components/nested_screens/home/InvestorScreen.js

```diff
@@ -228,6 +231,16 @@ export default function InvestorScreen(props) {
             color={THEME.icon.color.primary}
           />
         </TouchableOpacity>
+        <TouchableOpacity
```

Comment:
Could you add a check here for if its an AI investor? If it is then their should be no backtest button